src/model/Parameter.cpp

```diff
@@ -79,6 +95,29 @@ double Parameter::get(double time) {
     rtime = time;
   }
 
+  if (is_function == true) {
+    // Adapted from example from Basic Design example at http://www.partow.net/programming/exprtk/index.html
+    typedef double T;
+
+    typedef exprtk::symbol_table<T> symbol_table_t;
+    typedef exprtk::expression<T>   expression_t;
+    typedef exprtk::parser<T>       parser_t;
+
+    T t = T(time);
+
+    symbol_table_t symbol_table;
+    symbol_table.add_variable("t",t);
+
+    expression_t expression;
+    expression.register_symbol_table(symbol_table);
+
+    parser_t parser;
+
+    parser.compile(expression_string, expression);
```

Comment:
It is very easy for a user to add an incorrect math expression so you need to have a try/catch here and also throw an exception if `parser.compile()` returns false.